src/services/reviews.service.js

```diff
@@ -1,7 +1,16 @@
 import Reviews from '../database/models/reviews.model';
+import User from '../database/models/user.model';
 
 async function getReviews(productId) {
-  const reviews = await Reviews.findAll({ where: { productId } });
+  const reviews = await Reviews.findAll({
+    where: { productId },
+    include: [
+      {
+        model: User,
+        attributes: ['username'], // Specify the field(s) you want to retrieve
```

Comment:
you can include the user profile as well,  if he like first name and last name  and profile image is possible

Reply:
> you can include the user profile as well, if he like first name and last name and profile image is possible
<img width="826" alt="Screen Shot 2023-06-23 at 10 34 53" src="https://github.com/atlp-rwanda/Team-Sostene-E-commerce-bn/assets/103963968/454b075a-d8df-4937-9ba1-bb146426d7cc">
for the backend there is no option for adding user profile image and also names are displayed all at the same time as `names`

Reply:
- if no profile image, We need to have name only,  not birthday and telephone